src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow/sentryApplicationRowButtons.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import Access from 'app/components/acl/access';
+
+import {t} from 'app/locale';
+import {LightWeightOrganization, SentryApp, SentryAppInstallation} from 'app/types';
+import ActionButtons from './actionButtons';
+import {InstallButton, UninstallButton} from './installButtons';
```

Comment:
I think generally our components are put in separate files and exported as default. This makes it a bit easier to find the file based on the component name. We also wouldn't need to guess if we should be using default or named import.

Tbh I think it's also fine to have them in this file